sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -196,11 +225,29 @@ public synchronized void onActivityCreated(
 
   @Override
   public synchronized void onActivityStarted(final @NonNull Activity activity) {
+    if (performanceEnabled) {
+      ActivityFramesState.getInstance().addActivity(activity);
+    }
+
     addBreadcrumb(activity, "started");
   }
 
+  //  private boolean isHardwareAccelerated(Activity activity) {
+  //    // we can't observe frame rates for a non hardware accelerated view
+  //    return activity.getWindow() != null
+  //        && ((activity.getWindow().getAttributes().flags
+  //                & WindowManager.LayoutParams.FLAG_HARDWARE_ACCELERATED)
+  //            != 0);
+  //  }
```

Comment:
I read that reading slow and frozen frames are only possible if `FLAG_HARDWARE_ACCELERATED` is enabled, but it worked with it disabled during tests, so I'm not sure if I should keep this check or not.
`FLAG_HARDWARE_ACCELERATED` is disabled by default by the way

Reply:
Did you try it in a release build?

Reply:
yes, maybe thats the case for some OS versions only, I could not find anything though